app/src/main/res/layout/fragment_rewards_level.xml

```diff
@@ -64,15 +56,15 @@
       tools:text="Level 1"
       />
 
-  <include
-      android:id="@+id/rewards_progress_bar"
-      layout="@layout/rewards_progress_bar"
+  <com.asfoundation.wallet.promotions.GamificationProgressBarView
+      android:id="@+id/gamification_progress_bar"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:layout_marginStart="20dp"
       android:layout_marginTop="16dp"
       android:layout_marginEnd="20dp"
       android:layout_marginBottom="25dp"
+      android:elevation="2dp"
```

Comment:
The other usage of GamificationProgressBarView doesn't have elevation. Is it needed?

Reply:
It is needed